twitter/API/APIManager.m

```diff
@@ -65,6 +65,16 @@ - (void)getHomeTimelineWithCompletion:(void(^)(NSMutableArray *tweets, NSError *
        completion(nil, error);
    }];
 }
+- (void)postStatusWithText:(NSString *)text completion:(void (^)(tweet *, NSError *))completion{
+    NSString *urlString = @"1.1/statuses/update.json";
```

Comment:
This is more of a path really.

Reply:
I don't you. Can you elaborate?

Reply:
Sure! URLs have a scheme, a domain, a host, a path, and query parameters. If an entire url is https://www.google.com/foo then the part that’s “/foo” is called the path.